src/Microsoft.AspNetCore.Authentication/IClaimsTransformer.cs

```diff
@@ -6,8 +6,16 @@
 
 namespace Microsoft.AspNetCore.Authentication
 {
+    /// <summary>
+    /// Used for claims transformation.
+    /// </summary>
     public interface IClaimsTransformer
     {
+        /// <summary>
+        /// Transform a ClaimsPrincipal.
+        /// </summary>
+        /// <param name="principal"></param>
```

Comment:
Lazy :p 

Summary : Provides a central transformation point to change the specified principal.
Param: The principal to transform.
Returns: The transformed principal.